fourth-wall.css

```diff
@@ -81,6 +81,9 @@ body {
 #pulls li.thumbsup {
     background: #5c9317;
 }
+#pulls li.changes-requested {
+    background: #BF1E2D !important;
```

Comment:
Oh wow, yep that looks like it could be it. Not entirely sure how that's being pulled in. Must be something in the query string?